src/tests/issues.targets

```diff
@@ -2755,6 +2755,9 @@
         <ExcludeList Include = "$(XunitTestBinBase)/tracing/eventactivityidcontrol/eventactivityidcontrol/**">
             <Issue>needs triage</Issue>
         </ExcludeList>
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/Performance/CodeQuality/BenchmarksGame/k-nucleotide/k-nucleotide-9/**">
+            <Issue>https://github.com/dotnet/runtime/issues/67675</Issue>
+        </ExcludeList>
     </ItemGroup>
```

Comment:
should we remove this test exclusion from platform specific blocks in this file as part of this change or will you do that once we understand the actual issue ?

Reply:
I left it there, because I am not sure if there are separate issues blocking it on those platforms or what.